databricks/koalas/tests/test_dataframe.py

```diff
@@ -581,6 +581,17 @@ def test_clip(self):
         str_kdf = ks.DataFrame({'A': ['a', 'b', 'c']})
         self.assert_eq(str_kdf.clip(1, 3), str_kdf)
 
+    def test_binary_operators(self):
+        self.assertRaisesRegex(
+            ValueError,
+            'DataFrames is currently not supported',
+            lambda: ks.range(10).add(ks.range(10)))
+
+        self.assertRaisesRegex(
+            ValueError,
+            'DataFrame and sequence is currently not supported',
+            lambda: ks.range(10).add(ks.range(10).id))
```

Comment:
Does this mean we can't use `Series`, either?

Reply:
Yea. Series works differently with PySpark' for now..

```python
>>> import pandas as pd
>>> df = pd.DataFrame({'a': [1,2,3]})
>>> df + df.a
    a   0   1   2
0 NaN NaN NaN NaN
1 NaN NaN NaN NaN
2 NaN NaN NaN NaN
>>> df
   a
0  1
1  2
2  3
```